### ISSUE TITLE
about "Invalid port. Stopping..."

### ISSUE THREAD

**Issue by DahisC** [Arturo-1212/PTCGPB#17]
Hello, Arturo!
Thank you for developing such an amazing tool :D

This morning, I tried version 2.7, and it worked very smoothly.
However, after updating to version 2.8 this afternoon, I encountered an error message upon startup that says "Invalid port. Stopping."
I'm not sure if this issue is related to the update, or if I might have missed allowing the program through the firewall.
Could you kindly advise on the best way to resolve this issue?

(For now, I’ve tried turning off the firewall entirely, but the error message still persists.)

**Comment by Legendary108** [comment]
Hey @DahisC. I have the same error. I've tried everything I possibly could but nothing. Did you manage to solve it? If so, how?

**Comment by DahisC** [comment]
> Hey @DahisC. I have the same error. I've tried everything I possibly could but nothing. Did you manage to solve it? If so, how?

@Legendary108 

Hello! I just solved this problem.

It turns out that the issue was caused by me changing the installation path when I installed MuMu Player.
From the Script's log, I found that the port was trying to locate the folderPath.
The default path should be C:\Program Files\Netease (where MuMu Player stores its files),
but mine was installed on the D drive. Changing it to D:\Program Files\Netease solved the problem.

Give it a try! :D
